src/settings/Membership/validate.js

```diff
@@ -11,20 +11,42 @@ const validateMaxNameLength = (name = '') => (
   )
 );
 
+const validateMaxCodeLength = (code = '') => (
+  code.length > MAX_CODE_LENGTH && (
+    <FormattedMessage
+      id="ui-consortia-settings.settings.membership.error.codeExceedsLength"
+      values={{ count: MAX_CODE_LENGTH }}
+    />
+  )
+);
+
 const validateNameUniqueness = (name, i, items = []) => (
   items.some(({ name: _name }, _i) => (name === _name && i !== _i)) && (
     <FormattedMessage id="ui-consortia-settings.settings.membership.error.duplicate" />
   )
 );
 
-export const validate = ({ name }, i, items) => {
+const validateCodeUniqueness = (code, i, items = []) => (
+  items.some(({ code: _code }, _i) => (code === _code && i !== _i)) && (
```

Comment:
Since `code` [must be case-insensitive](https://issues.folio.org/browse/MODCON-26?focusedCommentId=167431&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-167431), I think it makes sense to coerce codes into one case.

Reply:
Probably also makes sense to add test case for that purpose